modules/apps/jaggery/emm/src/emm/units/platform-configuration/public/js/platform-configuration.js

```diff
@@ -591,6 +617,11 @@ $(document).ready(function () {
         var configOrgDisplayName = $("#ios-org-display-name").val();
         var iosLicense = tinymce.get('ios-eula').getContent();
 
+        fileNameMDMCert = getCertificate(fileNameMDMCert, iOSMDMName);
+        fileNameAPNSCert = getCertificate(fileNameAPNSCert, iOSAPNSName);
```

Comment:
Method name should be changed to something like "validateCertificateParams" since this validates not only the certificate.